src/handler-builder.ts

```diff
@@ -74,8 +83,24 @@ export class HandlerBuilder {
     };
   }
 
+  /**
+   * Create a handler for Realtime Database events.
+   * 
+   * `ref.onCreate` handles new data creation.
+   * `exports.myFunction = functions.handler.database.ref.onCreate((snap, context) => { ... })`
+   
```

Comment:
Do you think we'd want to add statements like the ones in the HTTP handler comments pointing out the similar-but-different relationship with parallel "regular"functions code?  

I don't feel sure we need to clarify that anywhere but the top level comment, but will respect your opinion :)

Reply:
HTTP functions was the only ones where the behavior was identical. (Since the trigger definition for an HTTP function normally doesn't include other metadata anyways, whereas for the background event types they do include things like database path, etc.)

To put it another, with the background functions, the handler namespace is "similar by different". With HTTP functions, the handler namespace is "identical" to the original functions. 

Do you think there's a way we can communicate that better?

Reply:
Ah OK, I didn't pick that up about HTTP functions.  I do think that just removing the "this is identical" phrasing is probably best.